frontends/web/src/components/password.tsx

```diff
@@ -316,7 +355,7 @@ const MatchesPattern = ({ regex, value = '', text }) => {
 
 const excludeKeys = /^(Shift|Alt|Backspace|CapsLock|Tab)$/i;
 
-const hasCaps = ({ key }) => {
+const hasCaps = ({ key }: { key: string }) => {
```

Comment:
not sure if better to use `{ key: string }` or just `KeyboardEvent`

Reply:
actually there is another bug with `event.shiftKey` at the end of this function, please remove both `@ts-ignore` and `eslint-disable-next-line` and check for a solution 🙏

Reply:
ah I think the whole function can just return `event.getModifierState('CapsLock')` then, the bug with that always returning false on qt was probably because `event` was not globally available and it was also not passed to the function.

Actually we can remove the function completely if `event.getModifierState` works, will test on qt now.

Reply:
Looks like adding the event listener does not even work when it is done in `UNSAFE_componentWilMount`! I'll remove the function and add it to `componentDidMount`

I noticed I can't really test the input dialog on qt, but I'll put the same logic in some other component just to test.

Reply:
Ok so `event.getModifierState('CapsLock')` still does not work on qt linux.

Reply:
yeah I believe @shonsirsha mentioned something like that before..